File: Python/Basic-to-Advance_Questions/factorial_number.py
def factorial():
    result = 1
    
    n = int(input('Enter number: '))
    
    if (n < 0):
        return 'Invalid Number please put positive numbers'
        
    
    for i in range(n, 0, -1):
        result *= i
    return result


factorial_number = factorial()
print(factorial_number)

File: JavaScript/Basic-to-Advance_Questions/factorialNumber.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});


rl.question('Enter positive integer Number: ', (num) => {
    num = +num;

    if (isNaN(num) || num <= 0 || !Number.isInteger(num)) {
        console.log(`Invalid input. Please enter a positive integer.`);
        rl.close();
        return;
    }

    let factor = 1;
    for (let i = 1; i <= num; i++) {
        factor *= i;
    }

    console.log(`Factorial of ${num} is ${factor}`);

    rl.close();
});
